src/pages/cart/components/coupon/coupon.js

```diff
@@ -0,0 +1,35 @@
+import {useCart} from "../../../../context/cart/cartContext"
+const Coupon = ()=>{
+  const {cartSummary} = useCart()
+    return (
+        <div className="cart-summary flex-col">
+        <p className="font-bold mb-1 text-xs">COUPONS</p>
+        <button className="button secondary-button-light coupon-btn">
+          <span>
+            <i className="fas fa-tag mr-1 icon-black text-xs"></i>
+          </span>
+          Apply Coupon
+        </button>
+        <div className="my-1 font-bold text-xs">PRICE DETAILS</div>
+        <div className="flex-row my-1 justify-between">
+          <p className="col-80-pc text-xs">Total MRP</p>
+          <p className="col-20-pc text-xs">₹{cartSummary.cartValue}</p>
```

Comment:
Suggestion: Can use optional chaining here for edge cases

Reply:
done !!